build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+pip install requirement.txt
```

Comment:
```suggestion
python -m pip install --requirement requirements.txt
```

At the moment this would fail as `pip install` needs the `-r`/`--requirement` flag to install from a `requirements.txt` file, and also the file that you have in your repo is `requirements.txt` not `requirement.txt`.

Reply:
Fixed it